dask/sizeof.py

```diff
@@ -208,3 +212,26 @@ def sizeof_pyarrow_table(table):
     @sizeof.register(pa.ChunkedArray)
     def sizeof_pyarrow_chunked_array(data):
         return int(_get_col_size(data)) + 1000
+
+
+def _register_entry_point_plugins(entry_points=None):
+    """Register sizeof implementations exposed by the entry_point mechanism."""
+    if entry_points is None:
+        entry_points = importlib.metadata.entry_points()
+
+    if sys.version_info >= (3, 10):
+        sizeof_entry_points = entry_points.select(group="dask.sizeof")
+    else:
+        sizeof_entry_points = entry_points.get("dask.sizeof", [])
```

Comment:
Why do we know this is Python 3.10 related as opposed to something else (like the `importlib_metadata` backport package being used)?

Reply:
I think copying the implementation from distributed would be best https://github.com/dask/distributed/blob/50d29110a430fe8457f73163082716844e1f11a6/distributed/comm/registry.py#L15

Reply:
Well I think part of the issue is this allows an `_EntryPoints` object to be passed in. Though maybe we could get rid of that functionality (if it is not needed) and move that code from Distributed into a utility function here and use in both libraries

Reply:
@graingert is there a particular reason for your suggestion? pyupgrade should catch this case too.